src/universal_agent/tgtg/config.py

```
@@ -0,0 +1,122 @@
+# HTTP POST target for stock / order / dead-item events.
+# Defaults to api.clearspringcg.com endpoint; leave blank to disable.
+TGTG_WEBHOOK_URL: str = os.getenv(
+    "TGTG_WEBHOOK_URL", "https://api.clearspringcg.com/tgtg/webhook"
+)
+
+# ── Telegram ──────────────────────────────────────────────────────────────────
+TELEGRAM_BOT_TOKEN: str = os.getenv("TELEGRAM_BOT_TOKEN", "")
+TELEGRAM_CHAT_ID: str = os.getenv("TELEGRAM_CHAT_ID", "")
+
+# ── Dashboard ─────────────────────────────────────────────────────────────────
+DASHBOARD_HOST: str = os.getenv("TGTG_DASHBOARD_HOST", "0.0.0.0")
```

Ruff (0.15.2)

[error] Possible binding to all interfaces

(S104)